#! /bin/sh /usr/share/dpatch/dpatch-run
## 25-strip-and-rpath.dpatch by Jens Peter Secher <jps@debian.org>
##
## DP: Strip neko executable prior to building rest of compiler tools and
## DP: use -rpath to force search in /usr/lib/neko.

@DPATCH@
diff -urNad neko-1.5.3~/Makefile neko-1.5.3/Makefile
--- neko-1.5.3~/Makefile	2007-01-28 22:34:39.000000000 +0100
+++ neko-1.5.3/Makefile	2007-02-21 23:51:10.847344645 +0100
@@ -7,7 +7,7 @@
 MAKESO = $(CC) -shared -WBsymbolic
 LIBNEKO_NAME = libneko.so
 LIBNEKO_LIBS = -ldl -lgc -lm
-NEKOVM_FLAGS = -Lbin -lneko
+NEKOVM_FLAGS = -Lbin -lneko -Wl,-rpath -Wl,/usr/lib/neko
 STD_NDLL_FLAGS = ${NEKOVM_FLAGS}
 INSTALL_FLAGS =
 RANLIB =
